tools/src/main/java/org/apache/kafka/trogdor/workload/ProduceBenchSpec.java

```diff
@@ -33,6 +33,9 @@
  * To configure a transactional producer, a #{@link TransactionGenerator} must be passed in.
  * Said generator works in lockstep with the producer by instructing it what action to take next in regards to a transaction.
  *
+ * You can also have the test refresh the partition list of the active topics at an interval by setting the
+ * `partitionRefreshRateMs` parameter to a positive number.  This is useful if you are adding partitions to the topics.
```

Comment:
My understanding is that `partitionRefreshRateMs` would need to be set to a positive number if we want producer worker to actually producer to new partitions. Is that correct? In this case, maybe we should say this in a comment vs. "This is useful"?

Reply:
Done.